resources/views/dropdown.blade.php

```diff
@@ -7,7 +7,16 @@
 ])
 
 <div
-    @if ($initAlpine ?? true) x-data="{ {{ $dropdownProperty }}: false }" @endif
+    @if ($initAlpine ?? true)
+        x-data="{ {{ $dropdownProperty }}: false }"
+        x-init="$watch('{{ $dropdownProperty }}', (expanded) => {
```

Comment:
The spatie responsive view use an `onload` event to replace the placeholder for the responsive images when the image is loaded, the problem is that it only works when the images are shown initially.

So images inside the dropdown (see navbar invitations that have the user avatar) will not load the responsive avatar until the dropdown is shown.

The lines here ensure to call `onload` event to every new responsive image inside a dropdown component



![image](https://user-images.githubusercontent.com/17262776/111203913-7bf68a00-85bd-11eb-9553-b24f0651683f.png)